src/ngraph/builder/split.cpp

```diff
@@ -74,3 +99,24 @@ NodeVector builder::split(const Output<Node>& value, size_t split_parts, int axi
     std::vector<size_t> length_parts(split_parts, length_axis_to_split / split_parts);
     return split(value, length_parts, axis_to_split);
 }
+
+NodeVector builder::opset1::split(const Output<Node>& value,
+                                  const std::vector<size_t>& length_parts,
+                                  int64_t axis)
+{
+    const auto axis_node = op::Constant::create(element::u64, Shape{}, {axis});
+    const auto length_parts_node =
+        op::Constant::create(element::u64, Shape{length_parts.size()}, length_parts);
```

Comment:
could you `use opset1` namespace for `Constant` operations?

Reply:
done